src/main/kotlin/com/cultureamp/kafka/connect/plugins/transforms/UnifyLegacySlackIntegrationPayload.kt

```diff
@@ -0,0 +1,104 @@
+package com.cultureamp.kafka.connect.plugins.transforms
+
+import org.apache.kafka.common.config.ConfigDef
+import org.apache.kafka.connect.connector.ConnectRecord
+import org.apache.kafka.connect.data.Field
+import org.apache.kafka.connect.data.Schema
+import org.apache.kafka.connect.data.SchemaBuilder
+import org.apache.kafka.connect.data.Struct
+import org.apache.kafka.connect.errors.DataException
+import org.apache.kafka.connect.transforms.Transformation
+import org.apache.kafka.connect.transforms.util.Requirements
+
+class UnifyLegacySlackIntegrationPayload<R : ConnectRecord<R>> : Transformation<R> {
```

Comment:
I feel adding xxxTransformation is too verbose??
Reason is when you are using it, it will be in the context of

```
"transforms": "UnifyLegacySlackIntegrationPayload",
"transforms.UnifyLegacySlackIntegrationPayload.type":"com.cultureamp.kafka.connect.transforms.UnifyLegacySlackIntegrationPayload",
```

I think it is pretty clear that it's a Transformer. Also, us sitting under the `/transforms` namespace

Reply:
Ok I see. The confusion was because we usually use `nouns` for class names. So may it could be `SlackIntegrationPayloadUnifier` or something similar, but that's ok for now. thanks.